### ISSUE TITLE
Tracking problems

### ISSUE THREAD

**Issue by TPRemRemillia** [ButzYung/SystemAnimatorOnline#97]
To explain. I use VRM models. And i decided to download this. I set everything up and the tracking was working. But then I closed vnyan to reopen it to check out the tracking with another model. When it opened, the only thing that would track was the arms and eyes. The body and the head would not move. I tried re-opening both, i tried redownloading both. Its still happening and i dont know what i did wrong? I need severe help.

**Comment by ButzYung** [comment]
Do you mean after restarting, the avatar on VNyan's side only has arms and eyes moving, while everything looks normal on XR Animator?

**Comment by TPRemRemillia** [comment]
Yes that is indeed what is happening. So im guessing it has something to do with vnyan. Have any idea on how to fix it?

**Comment by ButzYung** [comment]
Disable any webcam-based tracking on VNyan side and see if it works.
